src/coinbase/user.ts

```diff
@@ -35,7 +35,9 @@ export class User {
    * @returns the new Wallet
    */
   async createWallet(createWalletOptions: CreateWalletOptionsType = {}): Promise<Wallet> {
-    return Wallet.create(createWalletOptions);
+    const wallet = await Wallet.create(createWalletOptions);
+    await wallet.listAddresses();
```

Comment:
I don't think we should do this here fwiw.

`Wallet.create` and `user.createWallet` should have the exact same side-effects

Reply:
I was checking the output of createWallet, and I noticed we don't return the default address in the API output. So I thought we should include this call here. I'm happy to sync and share more details on this

Reply:
> and I noticed we don't return the default address in the API output.

We should be returning the full `Address` in the API response 🤔

Reply:
@alex-stone the type for the createWallet endpoint indicates that default_address might be optional as well

<img width="975" alt="Screenshot 2024-06-26 at 10 51 13 AM" src="https://github.com/coinbase/coinbase-sdk-nodejs/assets/797371/ba57cc35-00ec-4ae2-8569-1cf65b6a386a">

Reply:
Apologies, yes we do not create the address on the backend until we call the first `createAddress`!

However since we do call `createAddress` in this flow:
https://github.com/coinbase/coinbase-sdk-nodejs/blob/fa46b78058188ccaf8e1076f38c6d0b7badf8bed/src/coinbase/wallet.ts#L99

We should be updating our cached set of `addresses` and thus not need to load the addresses

Reply:
makes sense, I remove listAddresses usage.